swift-ticketing/test/swift_ticketing/client.clj

```diff
@@ -26,81 +39,85 @@
-                      app)]
+   (let [response (get-request (str "/event/" event-id))]
      {:path-params event-id
       :status (:status response)
       :response (response-to-json response)})))
 
 (defn create-tickets
   ([event-id request]
-   (let [{:keys [db-spec test-user-id]} fixtures/test-env
-         app (fn [req] ((swift-ticketing-app db-spec) req))
-         response (-> (post-request (str "/event/" event-id "/ticket") request)
-                      (mock/cookie "uid" test-user-id)
-                      app)]
+   (let [response (post-request (str "/event/" event-id "/ticket") request)]
      {:request request
       :status (:status response)
       :response (response-to-json response)})))
 
 (defn create-general-tickets
-  ([event-id] (create-tickets event-id (factory/general-ticket-request event-id))))
+  ([event-id] (create-tickets event-id (factory/general-ticket-request))))
 
 (defn create-seated-tickets
-  ([event-id] (create-tickets event-id (factory/seated-ticket-request event-id))))
+  ([event-id] (create-tickets event-id (factory/seated-ticket-request))))
 
 (defn reserve-ticket
   ([event-id request]
-   (let [{:keys [db-spec test-user-id]} fixtures/test-env
-         app (fn [req] ((swift-ticketing-app db-spec) req))
-         response (-> (post-request (str "/event/" event-id "/booking") request)
-                      (mock/cookie "uid" test-user-id)
-                      app)]
+   (let [response (post-request (str "/event/" event-id "/booking") request)]
      {:request request
       :status (:status response)
       :response (response-to-json response)})))
 
 (defn make-payment
   ([booking-id]
-   (let [{:keys [db-spec test-user-id]} fixtures/test-env
-         app (fn [req] ((swift-ticketing-app db-spec) req))
-         response (-> (post-request (str "/booking/" booking-id "/payment"))
-                      (mock/cookie "uid" test-user-id)
-                      app)]
+   (let [response (post-request (str "/booking/" booking-id "/payment") nil)]
+     {:path-params booking-id
+      :status (:status response)
+      :response (response-to-json response)})))
+
+(defn cancel-booking
+  ([booking-id]
+   (let [response (post-request (str "/booking/" booking-id "/cancel") nil)]
      {:path-params booking-id
       :status (:status response)
       :response (response-to-json response)})))
 
 (defn get-booking-status
   ([booking-id]
-   (let [{:keys [db-spec]} fixtures/test-env
-         app (fn [req] ((swift-ticketing-app db-spec) req))
-         response (-> (get-request (str "/booking/" booking-id "/status"))
-                      app)]
+   (let [response (get-request (str "/booking/" booking-id "/status"))]
+     {:path-params booking-id
+      :status (:status response)
+      :response (response-to-json response)})))
+
+(defn get-tickets
+  ([ticket-type-id]
+   (let [response (get-request "/ticket" {"ticket_type_id" ticket-type-id})]
+     (println response)
```

Comment:
There are a lot of print statements scaterred across the code (either commented or not). Please clean those up